Library/Formula/ghc.rb

```diff
@@ -36,9 +44,16 @@ class Ghc < Formula
     end
   end
 
-  resource "testsuite" do
-    url "https://downloads.haskell.org/~ghc/7.8.4/ghc-7.8.4-testsuite.tar.xz"
-    sha256 "d0332f30868dcd0e7d64d1444df05737d1f3cf4b09f9cfbfec95f8831ce42561"
+  if build.devel?
```

Comment:
Instead of a conditional, leave the resource as-is here, and place the devel resource in the `devel` block above.